components/form-switch/examples/00-default.js

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+
+import { FormSwitch } from '../src';
+
+export default () => (
+	<>
+		<h3>Switches without visible text label (Screenreader only text)</h3>
+		<FormSwitch name="ex1" id="ex1" toggleText={[]} srOnlyText>
+			Screen reader only text
+		</FormSwitch>
+		<FormSwitch name="ex2" id="ex2" srOnlyText>
+			Screen reader only text
+		</FormSwitch>
+		<FormSwitch name="ex3" id="ex3" toggleText={['Yes', 'No']} srOnlyText>
+			Screen reader only text
+		</FormSwitch>
+
+		<hr />
+
+		<h3>Switches with text label</h3>
+		<FormSwitch name="ex4" id="ex4">
+			Enable notifications
```

Comment:
For all "Enable notifications" label, the switch text could be "Yes/No"

Reply:
You're thinking about automating it? Enable... gets a yes/now. Turn... gets a on/off?

I think its best we don't lock any of this down. Let the dev implement what is required by the project on their end. We don't know what they're going to need and don't want to get in their way.

Reply:
Ooooooh are you just saying for the example? Yeah that's a good idea :)

Reply:
Yes just for example... and it kind of makes sense if you answer the question: "Do you want to enable the notifications?" Yes or No... On or Off seems weird in that context...